lib/Domain/StyleFixer.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Phpactor\CodeBuilder\Domain;
+
+final class StyleFixer
+{
+    /**
+     * @var StyleProposer[]
+     */
+    private $propsers;
+
+    public function __construct(StyleProposer ...$propsers)
+    {
+        $this->propsers = $propsers;
+    }
+
+    public function fix(string $code): string
+    {
+        foreach ($this->propsers as $proposer) {
+            $code = $proposer->propose($code)->apply($code);
+        }
+
+        return $code;
+    }
+
+    public function fixIntersection(string $code, TextEdits $previouslyAppliedChanges): string
```

Comment:
Added dedicated method for fixing an "intersectin" - we could add a command to run the above `fix` method on a whole file (don't know why, but useful for debugging at least).